sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -287,4 +427,10 @@ public EventDirectorySteps(
                         number.intValue(),
                         "Number of displayed cases is not correct")));
   }
+
+  private EventGroup collectEventGroup() {
```

Comment:
can we collect all data? in this way we can reuse this method for other tests, not only for one field needed here.
If we don't need to collect a POJO, then we can scrap data from this field and verify it directly in step

Reply:
removed it and moved collection of the Event group name to required location.